kabanero-cli-service/src/main/java/kabasec/JwtTracker.java

```diff
@@ -36,6 +54,8 @@ static String getSha256Digest(String in) {
         try {
             md = MessageDigest.getInstance("SHA-256");
         } catch (NoSuchAlgorithmException e) {
+            System.out.println("caught unexpected exception: "+ e);
```

Comment:
e.getMessage() and e.getCause()

Reply:
pretty sure printing e will just display the address of the object, not the contents

Reply:
e.toString will print the message, I'll add the cause